app/src/main/java/org/mozilla/focus/browser/integration/BrowserToolbarIntegration.kt

```diff
@@ -188,6 +226,9 @@ class BrowserToolbarIntegration(
         customTabsFeature?.stop()
         navigationButtonsIntegration?.stop()
         stopObserverSecurityIndicatorChanges()
+        toolbar.removeBrowserAction(tabsAction)
```

Comment:
I wonder why we have to remove the tab counter here? Shouldn't the observer handle all cases already? 🤔

Reply:
Is still added every time that's why I chose to remove it at stop()